neuro_san/http_sidecar/logging/http_logger.py

```diff
@@ -33,19 +33,19 @@ def __init__(self, forwarded_metadata: Sequence[str]):
         """
         Constructor
         """
+        # Initialize minimal metadata dictionary to contain some value
+        # for each metadata key we expect to be used.
+        self.base_metadata: Dict[str, Any] = {}
+        for key in forwarded_metadata:
+            self.base_metadata[key] = "None"
+        self.base_metadata["source"] = HttpLogger.HTTP_LOGGER_NAME
         LogContextFilter.set_log_context()
         self.setup_logging()
         # For our Http server, we have separate logging setup,
         # because things like "user_id" and "request_id"
         # can only be extracted and used on per-request basis.
         # Async Http server is single-threaded.
         self.logger = logging.getLogger(HttpLogger.HTTP_LOGGER_NAME)
-        # Initialize minimal metadata dictionary to contain some value
-        # for each metadata key we expect to be used.
-        self.base_metadata: Dict[str, Any] = {}
-        for key in forwarded_metadata:
-            self.base_metadata[key] = "None"
-        self.base_metadata["source"] = HttpLogger.HTTP_LOGGER_NAME
 
```

Comment:
Move this code up so that self.base_metadata is visible in setup_logging() scope.